lizard_connector/connector.py

```diff
@@ -241,17 +244,27 @@ def __init__(self, endpoint, base="https://demo.lizard.net", **kwargs):
         base = base.strip(r'/')
         if not base.startswith('https') and 'localhost' not in base:
             raise InvalidUrlError('base should start with https')
-        base = urljoin(base, 'api/v2') + "/"
-        self.base_url = urljoin(base, self.endpoint) + "/"
+        base = urljoin(base, 'api/v') + str(version) + "/"
+        self.base_url = urljoin(base, self.endpoint)
+        self.base_url += "/" if not self.base_url.endswith('/') else ""
```

Comment:
Terniary operator terwijl de else functionaliteit niets doet, dat is duidelijker te schrijven als `if not self.base_url.endswith('/'): self.base_url += '/'`, al dan niet op twee regels.